var/spack/repos/builtin/packages/likwid/package.py

```diff
@@ -41,6 +42,10 @@ class Likwid(Package):
     depends_on('perl', type=('build', 'run'))
 
     supported_compilers = {'clang': 'CLANG', 'gcc': 'GCC', 'intel': 'ICC'}
+    if platform.machine() == 'aarch64':
+        supported_compilers = {'gcc': 'GCCARMv8'}
+    elif platform.machine().startswith('ppc64'):
+        supported_compilers = {'gcc': 'GCCPOWER'}
```

Comment:
This doesn't need to be a class level attribute and can be moved inside the `install` method, unless I'm missing something.

Reply:
Done